### DISCUSSION TITLE
FDroid

### DISCUSSION THREAD

**Discussion by PeterKW** [TrianguloY/URLCheck#344] [General]
Can FDroid be updated? https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/com.trianguloy.urlchecker.yml The app is trelling me to update but there isn't anything to update?

**Comment by TrianguloY** [comment]
F-droid builds are automatically created, I don't have control over it. It usually takes 2-4 days.

Looks like the app is already built: https://monitor.f-droid.org/builds/log/com.trianguloy.urlchecker/32
It will be available to download probably today/tomorrow.
If f-droid is telling you to update something that f-droid still doesn't have...that's a bug on f-droid I'm afraid.
